follower.py

```diff
@@ -117,7 +117,7 @@ def getUserFollowers(user):
 # returns a list with all images to be displayed for a user
 def getImagesToShow(user):
     username = user
-    imagelist = getListFromCSV('data/userimages.csv')
+    imagelist = getListFromCSV('data/userImages.csv')
```

Comment:
Will this affect how the app currently runs?

Reply:
This change was needed for Travis to understand which file he needs to read from since he is case-sensitive. If this causes problems on your end we can meet and figure out a solution!

Reply:
In some methods in follower.py you used "userimages" and in others you used "userImages" so I don't think it makes a difference.